hw/top_earlgrey/doc/_index.md

```diff
@@ -16,24 +16,33 @@ This is not a specification of the final implementation.
   - DM (debug module)
   - PLIC (platform level interrupt controller)
   - U/M execution modes (user/machine)
+  - Enhanced Physical Memory Protection (ePMP)
 - Memory contents
-  - 512kB emulated eFlash for code and data storage
-  - 64kB SRAM for data storage
-  - 16kB ROM for secure boot code storage
+  - 1024kB eFlash
```

Comment:
should we mention that the various memories are scrambled?

Reply:
Done.